<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerferenceDialog</class>
 <widget class="QDialog" name="PerferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>9</y>
     <width>460</width>
     <height>332</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>460</width>
     <height>332</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>460</width>
     <height>332</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="generalPage">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>32</y>
       <width>261</width>
       <height>96</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>58</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="showReadmeCheck">
        <property name="text">
         <string>Show Readme At Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>58</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="showLineNumberCheck">
        <property name="text">
         <string>Show Line Number</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>58</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Max Recent Files:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxRecentFileSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="editorPage">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Font</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Family:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFontComboBox" name="fontFamilyComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="fontSizeSpinBox">
          <property name="minimum">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Style:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="fontStyleComboBox">
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bold</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Italic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bold Italic</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Tabs and Indentation</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="tabIndentsCheck">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Indents via Tabs</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>58</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2_3">
            <property name="text">
             <string>Indent Size:</string>
            </property>
            <property name="buddy">
             <cstring>indentSizeSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="indentSizeSpinBox">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Bitstream Charter</family>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> char</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="whitespacesCheck">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Show Spaces and Tabs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="backUnindentCheck">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Backspace Unindentation</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="spaceTabsCheck">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Use Spaces instead of Tabs</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="autoIndentCheck">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Auto-indentation</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>58</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2_4">
            <property name="text">
             <string>Tab Size:</string>
            </property>
            <property name="buddy">
             <cstring>tabSizeSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tabSizeSpinBox">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="suffix">
             <string> char</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>292</width>
       <height>132</height>
      </rect>
     </property>
     <property name="title">
      <string>Find/Replace Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Max Find/Replace History:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="maxHistorySpinBox"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="matchCaseCheck">
          <property name="text">
           <string>Match Case</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="regExpCheck">
          <property name="text">
           <string>User Regular Expression</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>121</width>
     <height>332</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>121</width>
     <height>332</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>121</width>
     <height>332</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>General</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Editor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Find/Replace</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="acceptButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
